src/kubectl-helm-minikube/install.sh

```diff
@@ -159,7 +159,7 @@ fi
 get_previous_version() {
     repo_url=$1
     # this would del the assets key and then get the second encountered tag_name's value from the filtered array of objects
-    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[1].tag_name' 
+    curl -s "$repo_url" | jq -r 'del(.[].assets) | .[0].tag_name' 
```

Comment:
Can we bump patch version of this Feature?

As a rule of thumb, whenever you contribute changes to any Feature, you always need to bump the Feature versions. Follow semantic versioning rules to determine the version bump.

Failing the bump version will lead into not releasing a new version of the Feature with these changes.

Reply:
made changes to the file